ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/ClientImpl.java

```diff
@@ -136,13 +153,19 @@ public BatchedQueryResult executeQuery(
       final String sql,
       final Map<String, Object> properties
   ) {
+    if (ConsistencyOffsetVector.isConsistencyVectorEnabled(requestProperties)) {
```

Comment:
Are you meaning to check this using `properties` rather than `requestProperties` since it's defined in `KsqlConfig`?

Reply:
Added to properties